client/gutenboarding/devtools.ts

```diff
@@ -18,3 +18,10 @@ export const wpDataDebugMiddleware: PageJS.Callback = ( context, next ) => {
 	}
 	next();
 };
+
+/**
+ * Function to use instead of `__()` from `@wordpress/i18n`.
+ *
+ * This is to avoid sending strings for translation tool too early.
```

Comment:
TODO; needs a valid JSdoc says linter

Reply:
We need to update or drop the jsdoc lint rules.

Reply:
If you add a space after the comment block, the lint issue may go away.

We'll want all the `_x`, `_n`, etc. family I think.

Reply:
Space helped 👍